tests/test_prompt_parser_validation.py

```
@@ -0,0 +1,121 @@
+        
+        # Test that parser can extract scores from this format
+        score1 = orchestrator._parse_hypothesis_scores(mock_response, 1)
+        score2 = orchestrator._parse_hypothesis_scores(mock_response, 2)
+        
+        # Verify scores were parsed correctly (not defaults)
+        assert score1.novelty == 0.7, f"Expected 0.7, got {score1.novelty}"
+        assert score1.impact == 0.8, f"Expected 0.8, got {score1.impact}"
+        assert score2.novelty == 0.5, f"Expected 0.5, got {score2.novelty}"
+        assert score2.impact == 0.6, f"Expected 0.6, got {score2.impact}"
+
+    def test_parser_handles_prompt_format_variations(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_parser_handles_prompt_format_variations`

Add return type annotation: `None`

(ANN201)